app/src/main/java/tech/ula/model/state/AppsStartupFsm.kt

```diff
@@ -156,6 +157,8 @@ class AppsStartupFsm(
         state.postValue(SyncingDatabaseEntries)
         appSession.filesystemId = appsFilesystem.id
         appSession.filesystemName = appsFilesystem.name
+        appSession.serviceType = appSession.serviceType
```

Comment:
This line looks unnecessary.

Reply:
Yes.  I was restoring the code that was there, more or less, before 2.6.1, but I obviously created something redundant here... lol